src/breeze/pages/requests/show_page.cr

```diff
@@ -102,4 +102,24 @@ class Breeze::Requests::ShowPage < BreezeLayout
       end
     end
   end
+
+  def render_query_row(query : BreezeSqlStatement)
+    link_styles = "block sm:border-t sm:border-gray-200 hover:bg-indigo-50 focus:outline-none focus:bg-gray-50 transition duration-150 ease-in-out"
+
+    link to: Queries::Show.with(query.id), class: link_styles do
+      div class: "flex items-center px-4 py-4 sm:px-4" do
+        div class: "min-w-0 flex-1 flex items-center" do
+          div class: "min-w-0 flex-1 px-4 sm:grid grid-cols-3 md:grid-cols-3 gap-4" do
+            div class: "text-sm leading-5 font-medium text-gray-500" { text "#{query.model}Query" }
```

Comment:
I didn't realize we don't pass along the query object (to be fair, I know almost nothing about breeze though 😄)

Reply:
Yeah, so this one actually comes all the way from [here](https://github.com/luckyframework/avram/blob/master/src/avram/queryable.cr#L213) where the `queryable` is basically the name of the model, and not the query class itself. 

I'm not a big fan of writing it like that since we can't guarantee that it's always going to be that, but I think it'll suffice for the 90% use case. We would have to update Avram, I think, in order to pass down what the query object running this would be.